VERSION

```diff
@@ -1 +1 @@
-2.5.0-rc1
+2.5.0-snapshot1
```

Comment:
Do we actually need to change the NCS version? We haven't done this so far for snapshots.